lib/facility_generator.rb

```diff
@@ -0,0 +1,45 @@
+class FacilityGenerator
+  def create_facility(facility)
+    facility.map do |data|
+      if data[:state] == "CO"
+        co_facility(data)
+      elsif data[:state] == "NY"
+        ny_facility(data)
+      elsif data[:state] == "MO"
+        mo_facility(data)
+      end
+    end
```

Comment:
Nice job with the helper methods here, definitely makes it easier to read and debug.

Reply:
Yeah, this class looks good, nice job.